runtime-modules/content/src/tests/mock.rs

```diff
@@ -815,7 +833,10 @@ impl MemberOriginValidator<Origin, u64, u128> for TestMemberships {
 
     fn is_member_controller_account(member_id: &u64, account_id: &u128) -> bool {
         match *member_id {
-            DEFAULT_MEMBER_ID => *account_id == DEFAULT_MEMBER_ACCOUNT_ID,
+            DEFAULT_MEMBER_ID => {
```

Comment:
If you mean why I allowed 2 accounts to act as controller of `DEFAULT_MEMBER_ID` member: It's because this way I can test if the bloat bond is returned to the correct account when I delete a channel/video. For example: I can sign `create_channel` with `DEFAULT_MEMBER_ACCOUNT_ID` and then delete channel using `DEFAULT_MEMBER_ALT_ACCOUNT_ID`.